Tasks/KubernetesManifestV0/task.json

```diff
@@ -88,7 +87,7 @@
             "defaultValue": 0,
             "visibleRule": "strategy = Canary && action = deploy",
             "validation": {
-                "expression": "isMatch(value, '(^(([0-9]|[1-9][0-9]|100)(\\.\\d{1,2})?)$)','Multiline')",
+                "expression": "isMatch(value, '(^(([0-9]|[1-9][0-9]|100)(\\.\\d*)?)$)','Multiline')",
```

Comment:
Why is this change required? is it blocking 100?

Reply:
It's not blocking 100. But, the previous rule did not allow more than 2 digits after decimal. So values like 95.125 won't be accepted.